sdk/src/wallet/core/operations/address_generation.rs

```diff
@@ -25,7 +25,7 @@ impl Wallet {
         address_index: u32,
         options: impl Into<Option<GenerateAddressOptions>> + Send,
     ) -> crate::wallet::Result<Ed25519Address> {
-        // TODO #1279: not sure yet whether we also should allow this method to generate addresses for different bip
+        // TODO: not sure yet whether we also should allow this method to generate addresses for different bip
```

Comment:
Is this not linked to an issue anymore?

Reply:
Not sure why the TODO is modified here, but the whole file is deleted in https://github.com/iotaledger/iota-sdk/pull/1586

Reply:
Ah, right, let's ignore then

Reply:
no, it isn't ... it was something that I though I could figure out during the "single account wallet" PR. It's not relevant anymore here as this file gets removed with https://github.com/iotaledger/iota-sdk/pull/1586. For generating addresses on the secret manager there's no reason to debate this.